.gitlocalize.yml

```diff
@@ -1,8 +1,18 @@
 pretranslate:
   segments:
+    ignore_frontmatter_keys:
+    - "layout"
```

Comment:
Hi Ilya,

Is it now a free form list, e.g. we can add freely our own field names in future?

Reply:
Hi Ewa, correct, we decided to implement this via the config to ensure more flexibility in case you'll want to add other frontmatter keys in future.

Reply:
Awesome 👍